src/components/Movie/Movie.js

```diff
@@ -26,6 +49,7 @@ class Movie extends Component {
             {this.props.unselectMovie()}}>back</button></Link>
         </div>
         <div className="info-container">
+        {!!this.state.message && <p>{this.state.message}</p>}
```

Comment:
Lets add a classname to this and also perhaps rename this to be more specific. What message?